docker/efficiency/builds.py

```diff
@@ -0,0 +1,221 @@
+import json
```

Comment:
`efficiency` seems a bit strange to me for a module name.  What do you think about just making this `docker.build` ?